web/v1/snowflake/sql-runner/playbooks/custom/02-page-views-join/01-page-views-join.yml.tmpl

```diff
@@ -0,0 +1,33 @@
+:targets:
+- :name:
+  :type:      snowflake
+  :account:
+  :database:
+  :warehouse:
+  :username:
+  :password:
+:variables:
+  :scratch_schema:     scratch
```

Comment:
I think the defaults should still have `_prod1` - we should follow the default that tech ops deploy for new customers. Double check with them if you're unsure.

Reply:
I do not doubt this may be the case. I had initially changed them so, and that is the reason i have this in its own commit, so that it can be dropped anytime.
The reason for reverting back started from the `derived` and `scratch` references in the tests' validation configs. I assumed (maybe wrongly) that it sounds more general to have `scratch` instead of e.g. `scratch_prod1`, since we only fill those variables to signify them being required, just like the `start_date`. 
Of course, this can be changed, i guess i'm not sure whether it being `scratch` or `scratch_prod1` creates a problem.

Reply:
Neither create a problem, it makes no difference when considering this project as a whole - but the default that we deploy have `_prod1` appended to the end, so using that in the playbooks reduces the amount of changes we need to make to deploy the model for customers. 

Really it's not a big deal, it's just nice if we have consistency across all of our defaults.

Reply:
Change of plan - we'll default to `atomic`, `scratch` and `derived` everywhere: https://github.com/snowplow/data-models/issues/41